website/blog/2024-04-05-release-1.9.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Podman Desktop 1.9 Release
+description: Podman Desktop 1.9 has been released!
+slug: podman-desktop-release-1.9
+authors: benoitf
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.9/juggling.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.9 Release! 🎉
+
+![Podman-desktop-1-9-hero](img/podman-desktop-release-1.9/juggling.png)
+
+We've got a new release with a ton of seal appeal! This release introduces:
```

Comment:
I don't have any better suggestions, but IMHO we've used the 'seal appeal' a little too often.

Reply:
changed